MetaMask/metamask-extension
# Move desktop code to flask build type

## Explanation

Move desktop support into the `flask` build type so it is included in Flask releases.

The `desktop` build type has intentionally not been deleted so any future desktop functionality can use it as a staging area until it is stable.

## Manual Testing Steps

`yarn start --build-type flask`

All desktop functionality should behave exactly the same but should now be present in Flask builds.

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
